### PR TITLE
Build on RHEL

### PR DESCRIPTION
**Pull request by jmelis**
Enable build on RHEL if TARGET=rhel

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @jmelis for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin - for example exclude certain file types so if PR contains only them it won't be checked.

**Comment by sbose78** [comment]
hi @jmelis , what needs to happen for the "WIP" tag to go? :)

File: Dockerfile.deploy.rhel [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM registry.devshift.net/osio-prod/base/golang:latest
+
+LABEL maintainer "Devtools <devtools@redhat.com>"
+LABEL author "Konrad Kleine <kkleine@redhat.com>"
```
Comment (alexeykazakov) [inline Dockerfile.deploy.rhel]: Well.. the author should be Jaime Melis I think ;)

Comment (kwk) [reply Dockerfile.deploy.rhel]: It is a copy of my Dockerfile ;) essentially

Comment (alexeykazakov) [reply Dockerfile.deploy.rhel]: It's not just a 100% copy, right? :) This docker file is based on your original file but with modifications. So, there are many authors of this file if to be precise...

File: cico_setup.sh [hunk 2]
```diff
@@ -102,26 +102,35 @@ function run_tests_with_coverage() {
 }
 
 function tag_push() {
-  TARGET=$1
-  docker tag fabric8-auth-deploy $TARGET
-  docker push $TARGET
+  local tag
```
Comment (kwk) [inline cico_setup.sh]: why not `local tag=$1` ?

Comment (jmelis) [reply cico_setup.sh]: it should have been like that, sorry I didn't see it before

Comment (kwk) [reply cico_setup.sh]: @jmelis no worries. I'm not even using `local` yet. Will implement it in the future in our shell scripts. ;)

**Pull request merged**